src/cinder/GeomIo.cpp

```diff
@@ -553,6 +552,24 @@ AttribSet Rect::getAvailableAttribs() const
 	return { Attrib::POSITION, Attrib::NORMAL, Attrib::TEX_COORD_0, Attrib::COLOR, Attrib::TANGENT };
 }
 
+void Rect::setDefaultColors()
+{
+	// upper-right, upper-left, lower-right, lower-left
+	mColors[0] = ColorAf( 0.0f, 1.0f, 0.0f, 1.0f );
```

Comment:
I personally find it more readable to use `0` and `1` here instead of `0.0f` and `1.0f`, which converts perfectly to floating point precision and is unambigious in these cases, but really not a big deal.